### ISSUE TITLE
Review Required Javascript Changes

### ISSUE THREAD

**Issue by simongamester** [PulseTile-Plugins/Plugin-Helm-PHR-Theme#3]
Hi @BogdanScherban 

I understand that Oleg has provided a number of javascript changes to be added to the Core repository in order to mirror the designs shown on the InVision project. I'm keen to make sure any changes to the Core repository are kept to a minimum - especially functional changes which should only apply to the Helm version.

Ideally all javascript, css and image changes related to Helm should be kept within this theme project and not Core unless those changes bring benefits to the whole platform.

In the first instance, I think we should look to get a functional version Helm which uses only Oleg's css and images, then look through the proposed javascript changes and make the following decisions for each:

1. Is there a CSS alternative?
2. Does it bring benefits to the Core project which justify it's inclusion in the Core project?
3. Can we leave this out of the project altogether and use the existing functionality instead?

Please could you identify each of javascript changes to this thread so we can review?

Simon

**Comment by BogdanScherban** [comment]
@simongamester 

Yes, I want to check all changes in JS, which he did for new styles.
Maybe, part of them should be provide in Core-part (and render by a some condition). 
Or opposite, we can create this changes automatically, when we create a build be generator.

It is necessary to check them and to decide, what way will be more optimal

**Comment by simongamester** [comment]
Thanks Bogdan. Could you post the changes here please?

**Comment by tony-shannon** [comment]
thanks @simongamester 

@BogdanScherban please point to those changes here for our review

@kbeloborodko again this is for your attention please

**Comment by PhilBarrett** [comment]
@simongamester - can this be closed now?
